.github/workflows/kind-dpu-offload.yml

```
@@ -24,6 +24,7 @@ env:
 
 jobs:
   kind-dpu-offload:
+    if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)